src/components/Tabs.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import classNames from 'classnames';
+
+interface Props {
+  tabs: Tab[],
+  selectedTab: Tab,
+  onTabSelected: (index: string) => void,
+}
+
+export const Tabs: React.FC<Props> = ({ tabs, selectedTab, onTabSelected }) => (
+  <div className="container">
+    {tabs.map(({ title, id }) => (
+      <button
+        key={id}
+        type="button"
+        className={classNames({
+          active: id === selectedTab.id,
+        })}
+        onClick={() => onTabSelected(id)}
```

Comment:
![image](https://user-images.githubusercontent.com/64437769/133434754-76f9b91f-1346-4623-8cca-0f34a346e567.png)

Reply:
I do not understand how to implement it

Reply:
you should check if current tab id is not the same as id of tab that was passed on props